src/uct/base/uct_cm.h

```diff
@@ -33,13 +33,18 @@ UCS_CLASS_DECLARE(uct_listener_t, uct_cm_h);
         (_status); \
     })
 
+
+#define UCT_CM_FAILURE_LOG(_cm, _fmt, ...) \
```

Comment:
maybe make it smallcase, to be consistent with other logging macros
eg `uct_cm_peer_error(...)`